src/mpl/include/mpl_base.h

```diff
@@ -98,6 +98,18 @@
 #define likely(x_)   (x_)
 #endif
 
+#ifdef HAVE_C11__STATIC_ASSERT
+#define MPL_Static_assert(cond_,msg_) _Static_assert(cond_,msg_)
```

Comment:
New additions to MPL have used all lowercase after `MPL_`. I think that would be preferred here.